pkg/azurefile/nodeserver.go

```diff
@@ -169,7 +169,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 
 	var mountOptions, sensitiveMountOptions []string
 	if protocol == nfs {
-		mountOptions = util.JoinMountOptions(mountFlags, []string{"vers=4,minorversion=1,sec=sys"})
```

Comment:
should make change here:
https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/1121ef184530c5e7b3ca2b51ad421830cf671e02/pkg/azurefile/nodeserver.go#L183

this mount option is only for cifs mount